src/swap/components/SwapProvider.tsx

```diff
@@ -93,6 +93,18 @@ export function SwapProvider({
     }
     // Emit Status
     onStatus?.(lifeCycleStatus);
+    // Reset lifecycle status after a success
+    if (
+      lifeCycleStatus.statusName !== 'init' &&
+      lifeCycleStatus.statusName === 'success'
+    ) {
+      setLifeCycleStatus({
```

Comment:
This is BAD, do not run `setLifeCycleStatus` inside this hook, it will potentially trigger an infit-loop.